Cython/Compiler/PyrexTypes.py

```diff
@@ -3802,7 +3805,7 @@ def create_from_py_utility_code(self, env):
             from .UtilityCode import CythonUtilityCode
             env.use_utility_code(CythonUtilityCode.load(
                 cls.replace('unordered_', '') + ".from_py", "CppConvert.pyx",
-                context=context, compiler_directives=env.directives))
```

Comment:
Currently breaks a few C++ tests - to be dealt with properly in https://github.com/cython/cython/pull/4206.